pkg/apis/v1alpha1/types.go

```diff
@@ -37,6 +37,9 @@ const (
 
 	RenewBeforeKey  = "csi.cert-manager.io/renew-before"
 	ReusePrivateKey = "csi.cert-manager.io/reuse-private-key"
+
+	KeystoreType = "csi.cert-manager.io/keystore-type"
+	KeystoreFile = "csi.cert-manager.io/keystore-file"
```

Comment:
Both the const names should end with Key for consistency (this is a key in the attributes map)